tests/test_modeling_common.py

```diff
@@ -30,12 +32,16 @@ def test_from_pretrained_save_pretrained(self):
 
         model = self.model_class(**init_dict)
         model.to(torch_device)
+        if isinstance(model, ModelMixin):
+            model._mps_warmup()
```

Comment:
Can we write a helper function right into the tests here to not have to add testing functionality to core `src/diffusers` (which would go into pip)